api_fhir/converters/claimConverter.py

```diff
@@ -245,47 +257,52 @@ def build_imis_information(cls, imis_claim, fhir_claim):
 
     @classmethod
     def build_fhir_string_information(cls, claim_information, code, value_string):
+        result = None
         if value_string:
             information_concept = ClaimInformation()
             information_concept.sequence = len(claim_information) + 1
             information_concept.category = cls.build_simple_codeable_concept(code)
             information_concept.valueString = value_string
             claim_information.append(information_concept)
+            result = information_concept
+        return result
 
     @classmethod
     def build_fhir_items(cls, fhir_claim, imis_claim):
-        fhir_items = []
-        cls.build_items_for_imis_item(fhir_items, imis_claim)
-        cls.build_items_for_imis_services(fhir_items, imis_claim)
-        fhir_claim.item = fhir_items
+        cls.build_items_for_imis_item(fhir_claim, imis_claim)
+        cls.build_items_for_imis_services(fhir_claim, imis_claim)
 
     @classmethod
-    def build_items_for_imis_item(cls, fhir_items, imis_claim):
+    def build_items_for_imis_item(cls, fhir_claim, imis_claim):
         for item in cls.get_imis_items_for_claim(imis_claim):
             if item.item:
                 type = Stu3ClaimConfig.get_fhir_claim_item_code()
-                cls.build_fhir_item(fhir_items, item.price_asked, item.qty_provided, item.item.code, type)
+                cls.build_fhir_item(fhir_claim, item.item.code, type, item)
 
     @classmethod
-    def build_items_for_imis_services(cls, fhir_items, imis_claim):
+    def build_items_for_imis_services(cls, fhir_claim, imis_claim):
         for service in cls.get_imis_services_for_claim(imis_claim):
             if service.service:
                 type = Stu3ClaimConfig.get_fhir_claim_service_code()
-                cls.build_fhir_item(fhir_items, service.price_asked, service.qty_provided, service.service.code, type)
+                cls.build_fhir_item(fhir_claim, service.service.code, type, service)
 
     @classmethod
-    def build_fhir_item(cls, fhir_items, price, quantity, code, item_type):
+    def build_fhir_item(cls, fhir_claim, code, item_type, item):
         fhir_item = FHIRClaimItem()
-        fhir_item.sequence = len(fhir_items) + 1
+        fhir_item.sequence = len(fhir_claim.item) + 1
```

Comment:
why you added 1, it is difficult to understand it from code perspective

Reply:
as above